EIPS/eip-1261.md

```diff
@@ -0,0 +1,246 @@
+---
+eip: 1261
```

Comment:
Why 1261?

Reply:
No reason.  It's the issue number. What do you recommend?